src/procedures/IssueTokens.ts

```diff
@@ -11,8 +11,16 @@ import {
 import { PolymathError } from '../PolymathError';
 import { Shareholder, SecurityToken } from '../entities';
 import { ModifyShareholderData } from './ModifyShareholderData';
+import { Factories } from '../Context';
 
-export class IssueTokens extends Procedure<IssueTokensProcedureArgs, Shareholder[]> {
+export const createRefreshSecurityTokenFactoryResolver = (
+  factories: Factories,
+  securityTokenId: string
+) => async () => {
+  return factories.securityTokenFactory.refresh(securityTokenId);
+};
+
+export class IssueTokens extends Procedure<IssueTokensProcedureArgs, Shareholder[] | void> {
```

Comment:
Why did you add `void` to the procedure's return type?

Reply:
Due to resolvers, I have found the right way to solve now